lib/analysis_server.dart

```diff
@@ -438,7 +438,7 @@ class _AnalysisServerWrapper extends Server {
 
     // Tracking `enableSuperMixins` here: github.com/dart-lang/sdk/issues/23772.
     analysis.updateOptions(new AnalysisOptions(
-      enableSuperMixins: true
+      //enableSuperMixins: true
```

Comment:
Looks like we don't need the entire analysis.updateOptions(...) line, so should I remove that as well? Or keep it with an "override options here" comment?